_scalingo_api/stacks.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Stacks
+layout: default
+---
+
+# Stacks
+
+--- row ---
+
+**Stack attributes**
+
+{:.table}
+| field       | type    | description                             |
+| ---         | ---     | ---                                     |
+| id          | string  | unique ID, starts with 'st-'            |
+| name        | string  | stack name display name                 |
+| base_image  | string  | docker image used to build your app     |
+| default     | boolean | is this the default stack for new app   |
+| created_at  | date    | creation date of the stack              |
+| description | string  | human readable description of the stack |
+
+||| col |||
+
+```json
+{
+  "id": "st-44b8ae3c-db5d-4e17-9344-65dcbfab095e",
+  "name": "scalingo-18",
+  "base_image": "scalingo/builder-18:v7",
+  "default": true,
+  "created_at": "2019-04-15T16:15:00.147Z",
+  "description": "Runtime stack based on Ubuntu 18:04"
```

Comment:
```suggestion
  "description": "Runtime stack based on Ubuntu 18.04"
```

And there are other places you made the same typo

Reply:
This typo was in prod data. Fixed in the 2 regions and in the article

Reply:
OK great, thanks :)